<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPassengersDialog</class>
 <widget class="QDialog" name="SearchPassengersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>573</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>جست‌وجوی سابقه مسافران</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/resources/icons/searchPassenger.png</normaloff>:/icons/resources/icons/searchPassenger.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="FLayoutSearchFields">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblId">
       <property name="text">
        <string>کد ملی</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>نام</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblFamily">
       <property name="text">
        <string>نام خانوادگی</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblFather">
       <property name="text">
        <string>نام پدر</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblPhone">
       <property name="text">
        <string>موبایل</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="HLayoutFromDate_2">
       <item>
        <widget class="QPushButton" name="btnSearchId">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/resources/icons/searchPassenger.png</normaloff>:/icons/resources/icons/searchPassenger.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtId"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtFamily">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtFather">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtPhone">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>جست‌وجوی سابقه سفر مسافران</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/resources/icons/searchTour.png</normaloff>:/icons/resources/icons/searchTour.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblTours">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>B Nazanin</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>کد اردو</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>مبدا</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>مقصد</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ظرفیت</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاریخ رفت</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>تاریخ برگشت</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>وضعیت اردو</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ثبت نام شده</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>خطا</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnReturn">
       <property name="text">
        <string>بازگشت</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/resources/icons/return.png</normaloff>:/icons/resources/icons/return.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
